### PR TITLE
Add benchmark for balances

### PR DESCRIPTION
**Pull request by clearloop**
This `benchmark` demand is original from #123, and relate to #323, we can review weights while benchmarking.

## Steps

+ [x] add benchmark command
+ [x] add benchmark for balances
  + skip `kton`, because it is as same as `ring`.
+ [x] patch docs for balances.
  + check the weight of balances

## Refs

1. [benchmarking command in substrate][0]
2. [benchmarking in substrate's balances][1]
3. [calcuating weights in substrate][2]

[0]: https://github.com/paritytech/substrate/blob/f5c6ebf966c126117717445416515987ecd13a40/frame/benchmarking/src/lib.rs#L28

[1]: https://github.com/paritytech/substrate/blob/master/frame/balances/src/benchmarking.rs

[2]: https://zhuanlan.zhihu.com/p/109696123?utm_source=wechat_session&utm_medium=social&utm_oi=52957873700864&s_s_i=oAd663Y%2F2YAtuV1kkjLQfZ0rqIrqYmI1JIQPvIPpgOU%3D&s_r=1&from=groupmessage&isappinstalled=0

### CODE REVIEW

**Comment by clearloop** [comment]
## Preview

```sh
 𝝺 ./target/debug/darwinia benchmark --extrinsic transfer --pallet ring
Pallet: "ring", Extrinsic: "transfer", Steps: [], Repeat: 1
u,e,extrinsic_time,storage_root_time
1,1000,5345000,1303000
100,1000,2984000,832000
199,1000,3037000,771000
298,1000,2165000,574000
397,1000,2062000,599000
496,1000,2045000,571000
595,1000,2207000,680000
694,1000,2089000,603000
793,1000,2354000,573000
892,1000,3013000,837000
991,1000,4462000,679000
1000,2,3918000,683000
1000,101,2369000,592000
1000,200,2401000,575000
1000,299,2397000,660000
1000,398,2436000,660000
1000,497,2779000,593000
1000,596,2105000,572000
1000,695,2551000,753000
1000,794,2474000,586000
1000,893,2950000,849000
1000,992,3158000,848000
Done.
```

The format of the `stdout` deserve a better `tui`, I'll do it after completing `benchmark` for all pallets.

File: bin/node/runtime/src/lib.rs [hunk 1]
```diff
@@ -643,7 +641,7 @@ construct_runtime!(
 		EthBacking: pallet_eth_backing::{Module, Call, Storage, Config<T>, Event<T>},
 		EthRelay: pallet_eth_relay::{Module, Call, Storage, Config<T>, Event<T>},
 		Kton: pallet_kton::{Module, Call, Storage, Config<T>, Event<T>},
-		Balances: pallet_ring::{Module, Call, Storage, Config<T>, Event<T>},
+		Ring: pallet_ring::{Module, Call, Storage, Config<T>, Event<T>},
```
Comment (HackFisher) [inline bin/node/runtime/src/lib.rs]: We should keep to use Balances as Module name, for easier compatibility with polkadot.js apps,

Refer https://github.com/darwinia-network/darwinia/pull/311

Comment (HackFisher) [reply bin/node/runtime/src/lib.rs]: ```Ring``` is our default Balances, so should still use ```Balances``` here.

Comment (clearloop) [reply bin/node/runtime/src/lib.rs]: Fixed.

**Pull request merged**